sdk/core/Azure.Core/tests/Azure.Core.Tests.csproj

```diff
@@ -23,13 +23,29 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\src\Shared\ArrayBufferWriter.cs" />
+    <Compile Include="..\src\Shared\AzureResourceProviderNamespaceAttribute.cs" />
+    <Compile Include="..\src\Shared\BufferedReadStream.cs" />
     <Compile Include="..\src\Shared\ConnectionString.cs" />
     <Compile Include="..\src\Shared\ForwardsClientCallsAttribute.cs" />
-    <Compile Include="..\src\Shared\RetriableStream.cs" />
+    <Compile Include="..\src\Shared\HashCodeCombiner.cs" />
+    <Compile Include="..\src\Shared\KeyValueAccumulator.cs" />
     <Compile Include="..\src\Shared\NoBodyResponse{T}.cs" />
     <Compile Include="..\src\Shared\OperationHelpers.cs" />
-    <Compile Include="..\src\Shared\AzureResourceProviderNamespaceAttribute.cs" />
+    <Compile Include="..\src\Shared\MultipartBoundary.cs" />
+    <Compile Include="..\src\Shared\MultipartContent.cs" />
+    <Compile Include="..\src\Shared\MultipartFormDataContent.cs" />
+    <Compile Include="..\src\Shared\MultipartReader.cs" />
+    <Compile Include="..\src\Shared\MultipartReaderStream.cs" />
+    <Compile Include="..\src\Shared\MultipartSection.cs" />
+    <Compile Include="..\src\Shared\RetriableStream.cs" />
+    <Compile Include="..\src\Shared\RequestContentContent.cs" />
+    <Compile Include="..\src\Shared\StreamHelperExtensions.cs" />
+    <Compile Include="..\src\Shared\StringSegment.cs" />
```

Comment:
I've played with the code a bit and think we can get rid of HashCodeCombiner, StringTokenizer, StringValues, ValueStringBuilder pretty easy